config/components/crd/bases/jobset.x-k8s.io_jobsets.yaml

```diff
@@ -9215,6 +9215,8 @@ spec:
                   TerminalState the state of the JobSet when it finishes execution.
                   It can be either Completed or Failed. Otherwise, it is empty by default.
                 type: string
+            required:
+            - restarts
```

Comment:
I don't think this is what you want. If this API has shipped, you cannot add newly required fields without this being a breaking change

Reply:
Thanks for calling this out!

yes this API is “shipper”. We are hoping to promote this API to v1 shortly so I don’t think we should have breaking changes.

Reply:
Understood and we don't want to break clients. Do you know how can we change this to be reported if it is 0, but not ask it be required? 

I removed omitempty from the annotation and use autogen to generate the schema, which changed the field to be required.